### ISSUE TITLE
Support for RN 0.30 zIndex

### ISSUE THREAD

**Issue by sodik82** [instea/react-native-popup-menu#7]
RN 0.30 now supports zIndex property that can be used to override default "zIndex strategy" in previous RN versions. This can lead to layout problems.

Pls add new `config` property to `MenuContext` that can influence zIndex of rendered backdrop and menu options. However for backward compatibility with previous RN versions there will be no default zIndex and will be activated only if user explicitly define desired zIndex.

**Comment by madox2** [comment]
Release of this feature is blocked by react native issues: https://github.com/facebook/react-native/issues/8970, https://github.com/facebook/react-native/issues/8968

**Comment by sodik82** [comment]
closing with won't fix since RN "z-index" is somehow different from web and it does not seem to have sense to use it in our situation ( see https://github.com/facebook/react-native/issues/8970 for more discussion)
